src/views/template/factory/factory.html

```diff
@@ -2,6 +2,12 @@
     <div class="panel-heading">
         {{#l}}Factory{{/l}}
     </div>
+
+    <button type="button" onclick="GAME.model.factorycontroller.increaseTruckSize(order)">
```

Comment:
In `factorycontroller`, add a 

```
$([your button]).click(function (e) {
 do stuff 
});
```

to the `registerEvent` method. `onclick` is something I'd discourage, as it mixes presentation with behaviour.